pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -296,6 +399,10 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.infeasible
+                    
+        # # WARNING
+        # unbounded='unbounded'                           # Demonstrated that problem is unbounded
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?